Source/DafnyRuntime/DafnyRuntime.cs

```diff
@@ -889,11 +897,11 @@ public abstract class Sequence<T> : ISequence<T> {
 
     public static ISequence<T> Create(BigInteger length, System.Func<BigInteger, T> init) {
       var len = (int)length;
```

Comment:
Should this be `checked` as well?

Reply:
Doesn't casting from biginteger to int raise an exception if it doesn't fit?